feature-proxy-manager/src/main/java/com/kinandcarta/create/proxytoggle/manager/model/Proxy.kt

```diff
@@ -1,10 +1,10 @@
 package com.kinandcarta.create.proxytoggle.manager.model
 
-data class Proxy(val address: String, val port: Int) {
+data class Proxy(val address: String, val port: String) {
```

Comment:
Any specific reason why?

Reply:
I had no validation before, and I was converting `.toInt()` and `.toString()` all the time.
In the end I just need a String, the user will input it as a String and I added validation, so there's no point in doing conversions all the time